packages/react-components/src/organisms/UserProfileGroups.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import { UserResponse } from '@asap-hub/model';
+import css from '@emotion/css';
+import { Card, Headline2, Paragraph, Link, Divider } from '../atoms';
+import { perRem, tabletScreen } from '../pixels';
+import * as colors from '../colors';
+
+const titleStyle = css({
+  fontWeight: 'bold',
+});
+
+const roleStyle = css({
+  color: colors.lead.rgb,
+});
+
+const containerStyles = css({
+  display: 'grid',
+
+  gridColumnGap: `${18 / perRem}em`,
+
+  margin: 0,
+  marginTop: `${24 / perRem}em`,
+
+  padding: 0,
+  listStyle: 'none',
+});
+
+const listItemStyle = css({
+  display: 'grid',
+
+  gridTemplateColumns: '1fr',
+  gridTemplateRows: '1fr 1fr',
+  gridRowGap: `${12 / perRem}em`,
+
+  [`@media (min-width: ${tabletScreen.min}px)`]: {
+    gridAutoFlow: 'column',
+    gridTemplateColumns: '1fr 1fr',
+
+    '&:not(:first-of-type)': {
+      gridTemplateRows: '1fr',
+    },
+
+    [`&:not(:first-of-type) > :nth-child(odd)`]: {
+      display: 'none',
+    },
+  },
+});
+
+type UserProfileGroupsProps = Pick<UserResponse, 'firstName'> & {
+  groups: ReadonlyArray<{
+    name: string;
+    role: 'Leader' | 'Member';
```

Comment:
It feels like this should be defined somewhere in model?

Reply:
I'm clarifying this with James. The designs don't use the same roles as the group page, that's why they differ.
Leader is someone that is a Chair or Project Manager in a group. Member is someone that is part of a team that is part of a group.

Reply:
Thanks for pointing this out. The requirement was to have the group roles extended with Member.
The "member" role only exists on the frontend, and it's inferred from the group response. Do you still believe the type definition should be elsewhere?